pkg/tasks/traffic/egress/yaml_configs.go

```diff
@@ -488,15 +487,29 @@ spec:
       weight: 100
   - match:
     - gateways:
-      - istio-egressgateway-sds
+      - istio-egressgateway
       port: 443
     route:
     - destination:
         host: my-nginx.mesh-external.svc.cluster.local
         port:
-          number: 8443
+          number: 443
       weight: 100
 `
+	meshExternalServiceEntry = `
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: mynginx-mesh-external
+spec:
+  hosts:
+  - my-nginx.mesh-external.svc.cluster.local
+  ports:
+  - number: 443
+    name: https
+    protocol: HTTPS
+  resolution: DNS
+`
```

Comment:
What about `location`? The documentation does not say what is the default value of this field, so could you add it explicitly?

Reply:
> What about `location`? The documentation does not say what is the default value of this field, so could you add it explicitly?

The namespace is the CP ns such as istio-system. Because ibm qe are testing a CP in another ns, I removed the namespace from the spec and use a parameter in the oc apply -n call.

Reply:
https://github.com/maistra/maistra-test-tool/pull/345/files#diff-6a15707b13a277d49843f7dcc4dcda5f1d67a62cc2fdd10f65a7256d1129e903R30
Thanks for reminding me and I will update the nginx example change in p and z example folders

Reply:
I'm not sure if I understand what you mean. I asked about `ServiceEntry.location`, because I expected it should be `MESH_EXTERNAL`, but you didn't specify it.

Reply:
I see. I misunderstood your question. Let me check and add it . Thanks, Jacek